src/pages/post/[postId]/index.tsx

```diff
@@ -20,78 +40,158 @@ export const getServerSideProps: GetServerSideProps<Props> = async ({
 })
 
 const PostDetailPage = ({ postId }: Props): ReactElement => {
-  const postQuery = useGetPostQuery(postId)
+  const getPostQuery = useGetPostQuery(postId)
+  const updateTradeStatusMutation = useUpdateTradeStatusMutation(postId)
+  const deletePostMutation = useDeletePostMutation(postId)
+  const router = useRouter()
+  const [tradeStatus, setTradeStatus] = useState<TradeStatusCodes>()
+  const [isOpenDialog, setIsOpenDialog] = useState(false)
+  const { isOpen, openModal, closeModal } = useModal()
```

Comment:
```suggestion
  const getPostQuery = useGetPostQuery(postId)
  const updateTradeStatusMutation = useUpdateTradeStatusMutation(postId)
  const deletePostMutation = useDeletePostMutation(postId)
  
  const [tradeStatus, setTradeStatus] = useState<TradeStatusCodes>()
  const [isOpenDialog, setIsOpenDialog] = useState(false)
  
  const { isOpen, openModal, closeModal } = useModal()
  const router = useRouter()
```
여기도 비슷한 훅끼리 묶어주는 게 좋을 것 같습니다!

Reply:
[완료 ! ](https://github.com/price-offer/offer-fe/pull/225/commits/6bdc4933fd3e32f96c98f5bb784e65ffd7b678dd#diff-d3e4a49efba75483b8c6b605e0129881440806cd0b2dc6b4e9e7b1c2860ea528R46-R51)